### DISCUSSION TITLE
Need the policy for critical only issyes

### DISCUSSION THREAD

**Discussion by karthikboncheruuvu** [advanced-security/policy-as-code#41] [Q&A] [answered]
@GeekMasher :

Hi ..Hope you are doing well.

i need ur suggestion on what to be used for codeql and dependabot policies to identify or break the loop for only critical issues.

we were using it as error, but it shows both critical and high..
i need only critical issues to break the loop

kindly suggest at the earliest..

**Comment by Pradoxzon** [comment, marked as answer]
The 'error' level will include all 'error', 'high', and 'critical' severity alerts by design. Could you just specify 'critical' instead of 'error' in your policy? That would limit it to only 'critical' alerts.

**Reply by GeekMasher** [reply]
@Pradoxzon is correct, you can set `codescanning` and `dependabot` severity levels to `critical` and it will only break on critical alerts. The [default policy is the following](https://github.com/advanced-security/policy-as-code/blob/74b09fc8e2bc145f11e5bb30104f87a2e7d70413/ghascompliance/defaults/policy.yml#L3-L8).

The [order is here](https://github.com/advanced-security/policy-as-code/blob/74b09fc8e2bc145f11e5bb30104f87a2e7d70413/ghascompliance/consts.py#L2-L15).

**Reply by karthikboncheruuvu** [reply]
it works
